home-open-api/src/main/resources/yaml/openapi.yaml

```diff
@@ -1107,6 +1111,410 @@ paths:
+      requestBody:
+        description: Create a new Invitation
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/CreateApartmentInvitation'
+        required: true
+      responses:
+        '201':
+          description: Successfully created
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ReadApartmentInvitation'
+        '400':
+          $ref: '#/components/responses/BadRequest'
+        "401":
+          $ref: '#/components/responses/Unauthorized'
+        "403":
+          $ref: '#/components/responses/Forbidden'
+        default:
+          $ref: '#/components/responses/InternalServerError'
+    get:
+      tags:
+        - apartment invitation
+      summary: "Get all invitations"
+      description: Get all invitations for apartment owners
+      operationId: queryInvitation
+      parameters:
+        - name: "apartment_id"
+          in: path
+          required: true
+          description: ID of the apartment
+          schema:
+            type: integer
+            format: int64
+            minimum: 1
+        - in: query
+          name: page_number
+          schema:
+            type: integer
+            minimum: 1
+            default: 1
+          required: false
+          description: The number of current page.
+        - in: query
+          name: page_size
+          schema:
+            type: integer
+            minimum: 1
+            maximum: 10
+            default: 5
+          required: false
+          description: The number of items to return.
+        - in: query
+          name: sort
+          schema:
+            type: string
+            default: id,asc
+        - in: query
+          name: filter
+          schema:
+            type: string
+        - in: query
+          name: id
+          schema:
+            type: integer
+            format: int64
+          example: 1
+        - in: query
+          name: email
+          schema:
+            type: string
+          example: example@gmail.com
+        - in: query
+          name: ownership_part
+          schema:
+            type: number
+            minimum: 0.0001
+            maximum: 1.0
+            multipleOf: 1e-4
+          example: 0.6588
+        - in: query
+          name: status
+          schema:
+            type: string
+          example: 1
+      responses:
+        '200':
+          description: Successful operation
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: "#/components/schemas/ReadApartmentInvitation"
+        '400':
+          $ref: '#/components/responses/BadRequest'
+        "401":
+          $ref: '#/components/responses/Unauthorized'
+        default:
+          $ref: '#/components/responses/InternalServerError'
+
+  /apartments/{apartment_id}/invitations/{id}:
+    patch:
```

Comment:
We are not going to implement nor patch nor put for an invitation. Only getOne and delete should be here. Delete can be accomplished only in allowed statuses.

Also we need to have an ability to manage owners - see them, delete, update (in terms of ownership area) but no create.

Reply:
> We are not going to implement nor patch nor put for an invitation. Only getOne and delete should be here. Delete can be accomplished only in allowed statuses.
> 
> Also we need to have an ability to manage owners - see them, delete, update (in terms of ownership area) but no create.

Proposed changes have been made